computation/src/main/java/com/powsybl/computation/ComputationParameters.java

```diff
@@ -30,24 +33,10 @@
      */
     OptionalLong getDeadline(String commandId);
 
-    Optional<String> getQos(String commandId);
-
+    /**
+     * @return an empty {@link ComputationParametersImpl}, but it supports plugin. See more {@link Extension}
```

Comment:
A detail: the fact that it returns a "ComputationParametersIMPL" is an implementation detail, I think it's better to only mention the interface here.